Packs/CommonTypes/IncidentFields/incidentfield-IP_Blocked_Status.json

```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "ipblockedstatus",
```

Comment:
Maybe IP Address Status? with the value of Blocked/Not Blocked?

Reply:
Given that we cannot know for sure that the IP was blocked, the value will be "Executed". maybe "IP Block" will be better